config/gulp/build.js

```diff
@@ -60,7 +60,7 @@ gulp.task('website', [ 'build', 'watch' ], function (done) {
     buildDrafts = '';
   }
 
-  var hugo = spawn('hugo', [ 'server', buildDrafts ]);
+  var hugo = spawn('hugo', [ 'server', buildDrafts, '--config=config_es.toml' ]);
```

Comment:
I love this idea of passing in the `--config=` param. Could we try something similar to the `SITE_BASEURL` that can get passed in via the environment before running `npm run build`, etc.